spec/feature/non_comp/board_grants_spec.rb

```diff
@@ -2,7 +2,10 @@
 
 feature "NonComp Board Grant Task Page", :postgres do
   before do
+    User.stub = user
+    nca_org.add_user(user)
     Timecop.freeze(post_ama_start_date)
+    FeatureToggle.enable!(:decision_review_queue_ssn_column)
   end
 
```

Comment:
```suggestion

after { FeatureToggle.disabled!(:decision_review_queue_ssn_column) }

```

Adding a disable in the after will prevent state from bleeding between tests.